.pipeline/scripts/release_notes_automation.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import os
+import re
+from datetime import datetime
+
+
+def read_file(file_name):
+    with open(file_name, 'r', encoding="utf8") as file:
+        return file.read()
+
+
+def write_file(file_name, data):
+    with open(file_name, 'w', encoding="utf8") as file:
+        file.write(data)
+
+unchanged_sections = [
+"""### 🚧 Known Issues
+
+-
+
+""",
+"""### 🔧 Compatibility Notes
+
+-
+
+""",
+"""### ✨ New Functionality
+
+-
+
+""",
+"""### 📈 Improvements
+
+-
+
+""",
+"""### 🐛 Fixed Issues
+
+- 
+"""]
+
+def remove_unchanged_sections(file, unchanged_sections):
+    for unchanged_section in unchanged_sections:
+        # if file contains unchanged_section, remove it
+        file = re.sub(unchanged_section, "", file)
+    return file
+
+def set_header(file, version):
+    date = datetime.today().strftime('%B %d, %Y')
+    # Replace the first line with: ## 5.2.0 - January 17, 2024
+    file = re.sub("^## .*", "## " + version + " - " + date, file)
+    return file
+
+def link_github_release(file, version):
+    old_github_release_link = "\[All Release Changes\]\(https://github.com/SAP/ai-sdk-java/releases/\)"
+    new_github_release_link = "[All Release Changes](https://github.com/SAP/cloud-sdk-java/releases/tag/rel%2F"+version+")"
+    file = re.sub(old_github_release_link, new_github_release_link, file)
+    return file
```

Comment:
(Question/Major)
This looks strange to me, why would there be a link to SAP Cloud SDK?

Reply:
You are correct, I missed to adapt this link when copying from Could SDK.